### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by denics** [OpenVoiceOS/ovos-stt-http-server#1]
Maybe it should be said somewhere that in order to pass the configuration to the used engine, the user should simply enlist in the configuration like:

```
# Speech to Text parameters
# Using Vosk as a provider for ovos-stt-plugin-server
  "stt": {
    "module": "ovos-stt-plugin-server",
    "fallback_module": "ovos-stt-plugin-vosk",
    "ovos-stt-plugin-server": {
      "url": "http://0.0.0.0:8080/stt",
      "model": "https://alphacephei.com/vosk/models/vosk-model-small-it-0.22.zip"
    },
    "ovos-stt-plugin-vosk": {
      "model": "https://alphacephei.com/vosk/models/vosk-model-small-it-0.22.zip"
    }
  },
  ```

and calling with ``ovos-stt-server --engine ovos-stt-plugin-vosk``

**Comment by rstropoli** [comment]
Agreed this is a very useful piece of information

**Comment by NeonDaniel** [comment]
Closed by #6
